spec/requests/breakers_integration_spec.rb

```diff
@@ -1,35 +1,50 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'support/rx_client_helpers'
 
 # TODO: possibly refactor this spec to be generic, not dependent on PrescriptionsController
 RSpec.describe 'Breakers Integration', type: :request do
```

Comment:
This is a really annoying spec, I wonder if we can follow the TODO here and refactor so it's generic instead of just using a different route